modules/ting_field_search/plugins/ting_field_search_ctools_export_ui.inc

```diff
@@ -158,11 +160,61 @@ function ting_field_search_profile_export_form(&$form, &$form_state) {
     '#default_value' => $profile->config['user_interaction']['alter_links'],
   );
 
+  $form['config']['user_interaction']['grouping'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Profile grouping'),
+    '#description' => t('Which group of search profiles should be shown when this profile is used.'),
+    '#default_value' => !empty($profile->config['user_interaction']['grouping']) ? $profile->config['user_interaction']['grouping'] : 'default',
+  );
+  $form['config']['user_interaction']['visibility'] = array(
+    '#type' => 'radios',
+    '#title' => t('Show block on specific pages'),
```

Comment:
What does "Block" refer to here? Perhaps a wee bit of copy/paste?

Reply:
Fixed :-)